solutions/CF Cellular Network.mdx

```diff
@@ -9,6 +9,58 @@ author: Benjamin Qi
 
 [Official Editorial](https://codeforces.com/blog/entry/46324?locale=en)
 
+This solution can also be implemented with Binary Search, but it's better to implement it with a set. The code below is provided only if you want additonal practice for Binary Searching a Sorted Array.
```

Comment:
Technically, I think "binary search" and "sorted array" should be lowercase. It's definitely fine as is though.